homeassistant/components/homekit/translations/en.json

```diff
@@ -1,25 +1,44 @@
 {
-    "options": {
+    "config": {
+        "abort": {
+            "port_name_in_use": "An accessory or bridge with the same name or port is already configured."
+        },
         "step": {
-            "yaml": {
-                "title": "Adjust HomeKit Options",
-                "description": "This entry is controlled via YAML"
+            "accessory_mode": {
+                "data": {
+                    "entity_id": "Entity"
+                },
+                "description": "Choose the entity to be included. In accessory mode, only a single entity is included.",
+                "title": "Select entity to be included"
             },
-            "init": {
+            "bridge_mode": {
                 "data": {
-                    "mode": "[%key:common::config_flow::data::mode%]",
-                    "include_domains": "[%key:component::homekit::config::step::user::data::include_domains%]"
+                    "include_domains": "Domains to include"
                 },
-                "description": "HomeKit can be configured expose a bridge or a single accessory. In accessory mode, only a single entity can be used. Accessory mode is required for media players with the TV device class to function properly. Entities in the \u201cDomains to include\u201d will be exposed to HomeKit. You will be able to select which entities to include or exclude from this list on the next screen.",
-                "title": "Select domains to expose."
+                "description": "Choose the domains to be included. All supported entities in the domain will be included.",
+                "title": "Select domains to be included"
             },
-            "include_exclude": {
+            "pairing": {
+                "description": "As soon as the {name} is ready, pairing will be available in \u201cNotifications\u201d as \u201cHomeKit Bridge Setup\u201d.",
+                "title": "Pair HomeKit"
+            },
+            "user": {
+                "data": {
+                    "mode": "Mode"
+                },
+                "description": "The HomeKit integration will allow you to access your Home Assistant entities in HomeKit. In bridge mode, HomeKit Bridges are limited to 150 accessories per instance including the bridge itself. If you wish to bridge more than the maximum number of accessories, it is recommended that you use multiple HomeKit bridges for different domains. Detailed entity configuration is only available via YAML. For best performance, and to prevent unexpected unavailability, create and pair a separate HomeKit instance in accessory mode for each tv media player and camera.",
```

Comment:
```suggestion
                "description": "The HomeKit integration will allow you to access your Home Assistant entities in HomeKit. In bridge mode, HomeKit Bridges are limited to 150 accessories per instance including the bridge itself. If you wish to bridge more than the maximum number of accessories, it is recommended that you use multiple HomeKit bridges for different domains. Detailed entity configuration is only available via YAML. For best performance, and to prevent unexpected unavailability, create and pair a separate HomeKit instance in accessory mode for each TV, media player and camera.",
```